demo-shell/src/app/components/search/search-bar.component.ts

```diff
@@ -27,23 +26,7 @@ import { AuthenticationService } from '@alfresco/adf-core';
 })
 export class SearchBarComponent {
 
-    @Input()
-    expandable: boolean = true;
```

Comment:
can you remove it also from the documentation?

Reply:
that's the demo shell wrapper component, cleaned up some properties left from previous refactorings